mathics/core/parser/tokeniser.py

```diff
@@ -230,7 +233,7 @@
     '<': ['Get', 'StringJoin', 'LessEqual', 'Less'],
     '=': ['SameQ', 'UnsameQ', 'Equal', 'Unset', 'Set'],
     '>': ['PutAppend', 'Put', 'GreaterEqual', 'Greater'],
-    '?': ['PatternTest'],
+    '?': ['Information', 'Definition','PatternTest'],
```

Comment:
Rather than adding an additional token we should reuse the `PatternTest` token for `Information`.

Reply:
Done

Reply:
Fixed